src/patterns/help-users-contact-a-department/all-contact-information/index.njk

```diff
@@ -0,0 +1,26 @@
+---
+title: All contact information
+layout: layout-example.njk
+---
+
+<p class="govuk-body govuk-!-font-weight-bold">Get help with your application</p>
```

Comment:
If this is a heading for the section I think you could consider using a HTML heading here with a heading class, this will make this section easier to navigate to in a screen reader.

For example:

`<h3 class="govuk-heading-m">Get help with your application</h3>`

Reply:
From a content point of view, is this heading generic enough for use in all services across government, does 'application' work for all services?

For example GOV.UK contact pages use a similar pattern to your proposal but would not be tied to a specific application necessarily depending on the service: https://www.gov.uk/government/organisations/hm-revenue-customs/contact/income-tax-enquiries-for-individuals-pensioners-and-employees

Reply:
Could have different title examples to resolve that ^

Reply:
Changes made as per suggestion.